### ISSUE TITLE
PHPCS fails when using PHP 8 Constructor property promotion

### ISSUE THREAD

**Issue by mehdibo** [squizlabs/PHP_CodeSniffer#3227]
**Describe the bug**
PHPCS fails when using PHP 8 Constructor property promotion
I'm using PSR2 rules

PHPCS displays :
```
Line indented incorrectly; expected 4 spaces, found 8
```

But when using 4 spaces I get:

```
 Multi-line function declaration not indented correctly; expected 8 spaces but found 4
```

**Code sample**
```php
class TestClass
{

    public function __construct(
        private int $a,
        private int $b,
        private int $c
    ) {
    }
}

```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs --standard=psr2 test.php`
3. See error message displayed
```
-------------------------------------------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 4 LINES
-------------------------------------------------------------------------------------------------------
 3 | ERROR | [ ] Each class must be in a namespace of at least one level (a top-level vendor name) 
 7 | ERROR | [x] Multi-line function declaration not indented correctly; expected 8 spaces but found 4
 8 | ERROR | [x] Multi-line function declaration not indented correctly; expected 8 spaces but found 4
 9 | ERROR | [x] Multi-line function declaration not indented correctly; expected 8 spaces but found 4
-------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------
```

**Running PHPCBF**
gives the following output:
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
test.php                                              FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 111ms; Memory: 8MB
```

**Expected behavior**
No errors

**Comment by jrfnl** [comment]
Duplicate of #3167 (and #3210) which has already been fixed. The fix will be included in the 3.6.0 release.

**Comment by mehdibo** [comment]
Sorry my bad, I tried searching for them, thanks!
